src/core/utils/fileutils.h

```diff
@@ -61,15 +61,23 @@ class QFIELD_CORE_EXPORT FileUtils : public QObject
     Q_INVOKABLE void addImageStamp( const QString &imagePath, const QString &text );
 
     static bool copyRecursively( const QString &sourceFolder, const QString &destFolder, QgsFeedback *feedback = nullptr, bool wipeDestFolder = true );
+
     /**
-     * Creates checksum of a file. Returns null QByteArray if cannot be calculated.
-     *
-     * @param fileName file name to get checksum of
-     * @param hashAlgorithm hash algorithm (md5, sha1, sha256 etc)
-     * @return QByteArray checksum
+     * Returns the checksum of a file. An empty QByteArray will be returned if it cannot be calculated.
+     * \param fileName file name to get checksum of
+     * \param hashAlgorithm hash algorithm (md5, sha1, sha256 etc)
+     * \return QByteArray checksum value
      */
     Q_INVOKABLE static QByteArray fileChecksum( const QString &fileName, const QCryptographicHash::Algorithm hashAlgorithm );
 
+    /**
+     * Returns an S3 ETag of a file. An empty string will be returned if it cannot be calculated.
```

Comment:
Is `S3` relevant (I thought ETag is a HTTP concept)

Reply:
@suricactus , ping.

Reply:
```suggestion
     * Returns an Object Storage (S3) ETag of a file. An empty string will be returned if it cannot be calculated.
```

It is relevant, as it is a implementation dependent. The E-Tag can be anything, it just happend that Amazon went with this implementation.

Reply:
@suricactus , should we rename the function fileS3ETag then?